config/config_exec/browserstack_conf/parallel.conf.js

```diff
@@ -1,12 +1,12 @@
-var browserstack = require('browserstack-local');
-
-const config = require('config');
+process.env["NODE_CONFIG_DIR"] ="./config/config_env";
+let config = require('config');
 var url = config.get("web.url");
 var globalTunnel = require('global-tunnel-ng');
-globalTunnel.initialize({
-  host: 'internetpu',
-  port: 8085,
+credential = require("../credentials.js")
 
+globalTunnel.initialize({
```

Comment:
look for the option where we don't necessarily need every time of execution

Reply:
It is only require for browser stack cloud